ui/src/config/section/storage.js

```diff
@@ -223,7 +223,7 @@ export default {
           label: 'label.change.offering.for.volume',
           args: ['id', 'diskofferingid', 'size', 'miniops', 'maxiops', 'automigrate'],
           dataView: true,
-          show: (record, store) => { return ['Allocated', 'Ready'].includes(record.state) && ['Admin'].includes(store.userInfo.roletype) },
+          show: (record, store) => { return ['Allocated', 'Ready'].includes(record.state) && (['Admin'].includes(store.userInfo.roletype) || !!store.apis.changeOfferingForVolume) },
```

Comment:
Should we remove the roletype check here?

Reply:
+1

normally the checks use
```
'listXXX' in store.getters.apis
```

Reply:
Hi @shwstppr ,
I really can't think of any scenario where checking for admin makes sense. 
I'll make a commit to remove the check.
thanks.

Reply:
Hi @weizhouapache ,
At first I tried to do it the way you explained, but I didn't succeed. So I left it at that.